src/mngt/ocf_mngt_flush.c

```diff
@@ -1035,6 +1035,9 @@ void ocf_mngt_cache_cleaning_set_policy(ocf_cache_t cache,
 	if (ocf_cache_is_standby(cache))
 		OCF_CMPL_RET(priv, -OCF_ERR_CACHE_STANDBY);
 
+	if (cache->conf_meta->cleaner_disabled)
+		OCF_CMPL_RET(priv, -OCF_ERR_CLEANER_DISABLED);
```

Comment:
requirements suggest, that it should be possible to set policy to NOP when cleaner is disabled. maybe move this check past line 1048.

Reply:
Sounds good. Done.